src/utils/contract.ts

```diff
@@ -1,8 +1,8 @@
 import { Interface, JsonFragment } from '@ethersproject/abi';
-import { Provider, RequestArguments, isNullAddress, EventData } from '../types';
-import FetchProvider from '../FetchProvider';
+import { RequestArguments, EventData } from '../types';
+import { Provider } from '../publicTypes';
+
 
-/** @internal */
```

Comment:
This one should not be public. We don't want to expose our own way of interacting with smart contracts.

Reply:
I started to whitelist the files I want jsdoc to pick up. Right now when I go on API reference page I can see only three classes. 
<img width="798" alt="Screen Shot 2020-09-30 at 10 59 18 AM" src="https://user-images.githubusercontent.com/16522024/94722328-01d29180-030c-11eb-9732-b4d1695acb44.png">